governance/votes/Executive vote - September 3, 2021.md

```diff
@@ -0,0 +1,101 @@
+
+123,333 DAI will be streamed to the **[GovAlpha (GOV-001) Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP3)** through September 30, 2021 at **0x01D26f8c5cC009868A4BF66E268c17B057fF7A73** in accordance with their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP11).
+
+103,134 DAI will be streamed to the **[Content Production (MKT-001) Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP5)** through October 31, 2021 at **0xDCAF2C84e1154c8DdD3203880e5db965bfF09B60** in accordance with their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP21).
+
+4,196,771 DAI will be streamed to the **[Oracles (ORA-001) Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP13)** through June 30, 2022 at **0x2d09B7b95f3F312ba6dDfB77bA6971786c5b50Cf** in accordance with their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP15).
+
+122,700 DAI will be streamed to the **[Governance Communications (COM-001) Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP8)** through December 31, 2021 at **0x1eE3ECa7aEF17D1e74eD7C447CcBA61aC76aDbA9** in accordance with their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP8).
+
+300,050 DAI will be streamed to the **[Growth (GRO-001) Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP4)** through October 31, 2021 at **0x7800C137A645c07132886539217ce192b9F0528e** in accordance with their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP19).
+
+4,080,000 DAI will be streamed to the **[Protocol Engineering (PE-001) Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP7)** through April 30, 2022 at **0xe2c16c308b843eD02B09156388Cb240cEd58C01c** in accordance with their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP7).
+
+**MKR Streams**
+
+1,051.25 MKR will be streamed to the **[Oracles (ORA-001) Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP13)** through June 30, 2022 at **0x2d09B7b95f3F312ba6dDfB77bA6971786c5b50Cf** in accordance with their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP15).
+
+803.1833 MKR will be streamed to the **[Growth (GRO-001) Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP4)** through July 1, 2022 at **0x7800C137A645c07132886539217ce192b9F0528e** in accordance with their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP19).
+
+**No more than** 15,422.5 MKR will be streamed to team members of the **[Protocol Engineering (PE-001) Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP7)** through April 30, 2025, at individual addresses maintained by the [Core Unit Facilitator](https://mips.makerdao.com/mips/details/MIP41c4SP7) according to the active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP7).
```

Comment:
I don't think we can say 'no more than' in here. We need to explain exactly what voting for this does, not leave it ambiguous. Like this proposal passing distributes x MKR, right? If they want to add additional recipients, that'll need to happen in a future executive vote.

Reply:
This should cover it, and it plainly visible in the code so we're not adding anything that could further unveil them